packages/tupaia-web/src/theme/constants.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+/**
+ * Theme constants that don't require being in the entire theme object, ie. things that won't change if a custom theme was applied
+ */
+
+export const CONTROL_BAR_PADDING = 10;
```

Comment:
I'm hoping that some of these consts can end up being in the component that they are used once we remove the context. and if not it would be good to move them out of the theme folder and into the consts folder